.github/workflows/presubmit.yml

```diff
@@ -134,22 +134,14 @@ jobs:
           name: thapi-bin
           path: thapi.tar
 
-  integration-tests:
+  Integration-tests:
```

Comment:
```suggestion
  integration-tests:
```

Reply:
All other step have start capitlaized

Reply:
Oh no but they show in the CI capilized... humm

Reply:
Wanted to avoid the lower-case in `https://github.com/argonne-lcf/THAPI/actions/runs/10080775871/job/27871485893` "Load Babeltrace" but then "integration test"

Reply:
Ok fixed in 3edff95